.github/workflows/pr.yaml

```diff
@@ -122,31 +126,32 @@ jobs:
   wheel-build-cpp:
     needs: checks
     secrets: inherit
-    uses: rapidsai/shared-workflows/.github/workflows/wheels-build.yaml@branch-25.04
+    uses: rapidsai/shared-workflows/.github/workflows/wheels-build.yaml@customize-matrix
     with:
       matrix_filter: group_by([.ARCH, (.CUDA_VER|split(".")|map(tonumber)|.[0])]) | map(max_by(.PY_VER|split(".")|map(tonumber)))
       build_type: pull-request
       script: ci/build_wheel_cpp.sh
   wheel-build-python:
     needs: wheel-build-cpp
     secrets: inherit
-    uses: rapidsai/shared-workflows/.github/workflows/wheels-build.yaml@branch-25.04
+    uses: rapidsai/shared-workflows/.github/workflows/wheels-build.yaml@customize-matrix
     with:
       build_type: pull-request
       script: ci/build_wheel_python.sh
   wheel-tests:
     needs: [wheel-build-python, changed-files]
     secrets: inherit
-    uses: rapidsai/shared-workflows/.github/workflows/wheels-test.yaml@branch-25.04
+    uses: rapidsai/shared-workflows/.github/workflows/wheels-test.yaml@customize-matrix
     if: fromJSON(needs.changed-files.outputs.changed_file_groups).test_python
     with:
+      # this should run the PR matrix of tests
```

Comment:
Yep, that worked.

<img width="330" alt="image" src="https://github.com/user-attachments/assets/0049d0a4-8f04-4ca3-9c04-d7086e02da43" />

([build link](https://github.com/rapidsai/rmm/actions/runs/13927555871?pr=1865))

Matches the 2 jobs variants in the `pull-request` matrix here:

https://github.com/rapidsai/shared-workflows/blob/4e9342ab26d251765017010e9471f434ab4c6627/.github/workflows/wheels-test.yaml#L81-L85